### DISCUSSION TITLE
React Native Support

### DISCUSSION THREAD

**Discussion by AhmedIsmail1994** [niklasvh/html2canvas#2690] [Q&A] [answered]
Hello All,
Can I use html2canvas with react native app and get specific View as Image?

Thanks

**Comment by niklasvh** [comment, marked as answer]
No, html2canvas cannot be used outside of a browser (such as in node.js or react native) as it depends on the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model).
